src/formatDate.js

```diff
@@ -65,9 +65,13 @@ function formatDate(date, fromFormat, toFormat) {
   fromFormat.forEach((part, index) => {
     if (part === 'YYYY' || part === 'YY') {
       year = dateParts[index];
-    } else if (part === 'MM') {
+    }
+
+    if (part === 'MM') {
       month = dateParts[index];
-    } else if (part === 'DD') {
+    }
+
+    if (part === 'DD') {
```

Comment:
I was talking abou switch structure here - generally you should use if's (without else if) only if there's return statements